looper/models.py

```diff
@@ -1126,24 +1127,40 @@ def as_data_frame(self):
         return _pd.DataFrame([s.as_series() for s in self.samples])
 
 
-    def to_csv(self, path):
+    def write(self, path, sep=None):
```

Comment:
I'd prefer to have just `write`, but only slightly. It's totally up to you; I'll add it back if you'd like.